assets/base/config.json

```diff
@@ -2,12 +2,12 @@
     "DefaultServerUrl": "",
     "TestServerUrl": "http://localhost:8065",
     "ShowErrorsList": false,
-    "EELearnURL": "about.mattermost.com",
+    "EELearnURL": "mattermost.com",
     "TeamEditionLearnURL": "mattermost.org",
-    "AboutTeamURL": "http://www.mattermost.org/",
+    "AboutTeamURL": "https://mattermost.com/",
     "AboutEnterpriseURL": "https://mattermost.com/",
     "PlatformNoticeURL": "https://mattermost.com/platform-notice-txt/",
```

Comment:
Hi @cwarnermm Not sure if this is within the scope of your changes but the `PlatformNoticeURL` doesn't seem to have the proper url (https://mattermost.com/platform-notice-txt/ is a non-existent page)

Reply:
@josephbaylon - It is and I've made that update. Thank you for flagging it!